conmon/conmon.c

```diff
@@ -139,6 +139,7 @@ static char *opt_cuuid = NULL;
 static char *opt_runtime_path = NULL;
 static char *opt_bundle_path = NULL;
 static char *opt_pid_file = NULL;
```

Comment:
I think we should rename this to `opt_container_pid_file` to match the new `--container-pidfile`.

Reply:
yes, good idea, I am going to change it